introduction.md

```diff
@@ -5,3 +5,4 @@ I kill the following:
 *Git
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?